plugins/main/public/components/agents/fim/inventory/fileDetail.tsx

```diff
@@ -357,7 +358,12 @@ export class FileDetails extends Component {
                       >
                         <EuiButtonIcon
                           onClick={() => {
-                            this.addFilter(item.field, value);
+                            this.addFilter(
+                              item.field,
+                              item.field === 'date' || item.field === 'mtime'
+                                ? rawValue
+                                : value,
```

Comment:
You have the option to create a new function called getValue, which can be designed to retrieve specific values as needed.